src/main/groovy/com/bugsnag/android/gradle/BugsnagUploadAbstractTask.groovy

```diff
@@ -111,6 +112,13 @@ abstract class BugsnagUploadAbstractTask extends BugsnagVariantOutputTask {
         if (project.bugsnag.overwrite || System.properties['bugsnag.overwrite']) {
             mpEntity.addPart("overwrite", new StringBody("true"))
         }
+
+        project.logger.info("apiKey: ${apiKey}")
+        project.logger.info("appId: ${applicationId}")
+        project.logger.info("versionCode: ${versionCode}")
+        project.logger.info("buildUUID: ${buildUUID}")
+        project.logger.info("versionName: ${versionName}")
+        project.logger.info("overwrite: ${project.bugsnag.overwrite}")
```

Comment:
Would this be needed at a debug level, rather than info? Its quite verbose.

Reply:
Yes - I've updated the log level.